Fundamentals/Setup/Upgrading/migrating-to-v8.md

```diff
@@ -32,6 +32,15 @@ The database types that are supported are SQL Server and SQL CE.
 Feedback from user testing has shown that some databases are harder to migrate than others.
 We are collecting [a list of these known issues on our GitHub Issue Tracker](https://github.com/umbraco/Umbraco-CMS/issues?utf8=%E2%9C%93&q=label%3Acategory%2Fcontent-migration+). There is a community package: [Pre-migration health checks](https://our.umbraco.com/packages/developer-tools/pre-migration-health-checks/) that you can install on your V7 site before migration to help identify and resolve some of these common issues before triggering the migration steps detailed below.
 
+:::note
+A migration was introduced in v8.6 which can break the migration process. See [Issue #7914](https://github.com/umbraco/Umbraco-CMS/issues/7914) for more details. 
+
+There are two ways to work around this issue:
+
+* Migrate to version 8.5 as a first step, and then post-migration, carry out a normal Umbraco upgrade to the latest version of Umbraco 8, or
```

Comment:
```suggestion
* Migrate to version 8.5 as a first step and then post-migration, carry out a normal Umbraco upgrade to the latest version of Umbraco 8, or
```

Reply:
We use a lot - and I mean, A LOT! - more commas in danish writing, so I also tend to add a bunch of them when writing english as well 😅 
Thanks for catching the rogue ones for me 😎